pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -315,13 +315,6 @@ func pickOneNodeForPreemption(nodesToVictims map[string]*extenderv1.Victims) str
 	var minNodes1 []string
 	lenNodes1 := 0
 	for node, victims := range nodesToVictims {
-		if len(victims.Pods) == 0 {
-			// We found a node that doesn't need any preemption. Return it!
-			// This should happen rarely when one or more pods are terminated between
-			// the time that scheduler tries to schedule the pod and the time that
-			// preemption logic tries to find nodes for preemption.
-			return node
-		}
```

Comment:
Why remove this?

Reply:
cause we cannnot got this situation anymore, the case as the comment states would be handled by the normal scheduling process instead of the preemption here.

Reply:
SG. I think this code was introduced long time ago, by then it was not guaranteed we have a consistent cluster view in one scheduling cycle.